doc/api/buffer.md

```diff
@@ -1078,8 +1078,8 @@ const u16 = new Uint16Array([0, 0xffff]);
 const buf = Buffer.copyBytesFrom(u16, 0, 1);
```

Comment:
I think the OP might have meant that instead, it seems marginally more helpful to understand how the API works
```suggestion
const buf = Buffer.copyBytesFrom(u16, 1, 1);
```

Reply:
Thank you for suggestion. Applied your suggestion. I found similar example under test directory(test/parallel/test-buffer-from.js:78).